code/controllers/subsystem/explosions.dm

```diff
@@ -459,16 +459,16 @@ This way we'll be able to draw the explosion's expansion path without having to
 		timer = TICK_USAGE_REAL
 		var/list/throw_turf = throwTurf
 		throwTurf = list()
-		for(var/t in throw_turf)
-			var/turf/affected_turf = t
+		for(var/turf/affected_turf AS in throw_turf)
 			if(QDELETED(affected_turf))
 				continue
-			for(var/am in affected_turf)
-				var/atom/movable/thing_to_throw = am
+			for(var/atom/movable/thing_to_throw AS in affected_turf)
 				if(thing_to_throw.anchored || thing_to_throw.move_resist == INFINITY)
 					continue
 
 				for(var/throw_source in throw_turf[affected_turf])
+					if(throw_turf[affected_turf][throw_source][3] < (thing_to_throw.move_resist * MOVE_FORCE_THROW_RATIO))
```

Comment:
I don't really get what the benefit or idea for having MFTR is but whatever. Do you want this to be <=?

Reply:
As mentioned on discord, < is fine. the ratio thing is ?? since its 1, but its used elsewhere so won't touch it.